src/nova-components/CreateFavorites/resources/js/components/DropdownInput.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="flex border-b border-40 remove-bottom-border">
+    <div class="w-1/5 px-8 py-6">
+      <label for="user" class="inline-block text-80 pt-2 leading-tight">
+        User <span class="text-danger text-sm">*</span></label
+      >
+    </div>
+    <div class="py-6 px-8 w-1/2">
+      <div class="flex items-center">
+        <select
+          data-testid="users-select"
+          dusk="user"
+          class="form-control form-select w-full"
+          :class="{ 'border-danger': error }"
+          v-bind:value="input"
+          v-on:input="$emit('input', $event.target.value)"
```

Comment:
don't use events to pass data to parent. hint: pass a function to the component as a prop